internal/api/http.go

```diff
@@ -187,8 +187,17 @@ func inspectableMIMEType(t string) bool {
 	return strings.HasPrefix(t, "text/") || jsonTypeRE.MatchString(t)
 }
 
-func isValidHost(reqHost, hrtHost string) bool {
-	return strings.Contains(reqHost, hrtHost)
+func isRequestHostValid(reqHost, hrtHost string) bool {
+	reqHost = strings.ToLower(reqHost)
+	hrtHost = strings.ToLower(hrtHost)
+
+	return (reqHost == hrtHost) ||
+		// // Check if the hrtHost is a subdomain of reqHost
+		// // e.g strings.HasSuffix("api.github.com", "".github.com") == true
```

Comment:
This comment is wrong and full of typos. You can remove it; it should be easy to understand what the logic is doing here.

Reply:
Whoops - the comments should have been removed. They were for my reference when debugging.